src/less/core.less

```diff
@@ -74,6 +74,11 @@
     width: 100%;
   }
 }
+
+.rw-now {
+  border: 1px solid #adadad;
```

Comment:
I think this is the only thing i'd change, the grey border feels to much like the selected, hover, and focus styles. Not that it matters that much since its a default style. Even still i think i'd like to differentiate it a bit, heres what I thought looked ok on my end, Mostly though it makes sure not to hardcode the color so less consumers can tweak as they wish.

```less
@calendar-now-color: darken(@state-border-focus, 15%); // in bootstrap-theme.less

.rw-now {
  &,
  &.rw-state-focus {
    color: @calendar-now-color;
    font-weight: 600;
  }
}
```

No pressure to change it yourself if you don't want (let me know), but i wouldn't want to tweak it without letting you know.